docs/docs/gallery.md

```diff
@@ -0,0 +1,7 @@
+# Description 
+
+QED III suite of simulations follow supernova-driven outflows in varying galactic environment. This movie shows the fiducial case of outflows emerging from a 1 kpc^2 patch of disc in the Solar neighbourhood. The gas surface density is 13 Msun/pc/pc and the SN-rate is 6e-5/yr. The box height is 8 kpc and the resolution is uniformly 2 pc in all directions. For cooling the gas metallicity has been fixed at Solar.
```

Comment:
Can you change the numbers and units to LaTeX formatting?

Reply:
Done.